runtime/doc/filetype.txt

```diff
@@ -513,40 +513,40 @@ Local mappings:
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
 View manpages in Nvim. Supports highlighting, autocompletion, locales, and
-navigation. See also |find-manpage|.
+navigation. Also see |find-manpage|.
 
 To use Nvim as a manpager:
 
      export MANPAGER="nvim -c 'set ft=man' -"
 
+All commands and mappings will attempt to reuse the closest man window
+(left/right) and otherwise create a horizontal split.
```

Comment:
I would leave out the word "horizontal" here, since that's the default.  Also, once the `<mods>` patch gets included from Vim, then the command should be updated to use that so the user could do `:vert Man ...` to get a vertical split.